src/App.tsx

```diff
@@ -15,35 +16,112 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+const SORT_FIELD_ALPHABETICAL = 'alphabetical';
+const SORT_FIELD_LENGTH = 'length';
```

Comment:
According to the task description, you should use an `enum` for sorting options instead of string literals. Consider defining an `enum` for `SortField`.